sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/implementation/models/BackupSecretResult.java

```diff
@@ -30,7 +30,7 @@ public BackupSecretResult() {}
      */
     public byte[] getValue() {
         if (this.value == null) {
-            return new byte[0];
+            return null;
```

Comment:
This is a break - albeit it better - change in behavior. Is this okay?

Reply:
setting `null-byte-array-maps-to-empty-array: true` in swagger/readme.md should revert this change (if we don't want this behavior break)

@alzimmermsft

Reply:
This was done as `new byte[0]` causes issues sometimes in logic that expected null.

Reply:
It makes more sense, but should it go into the CHANGELOG.md? Not blocking, and given your explanation of the other concerns, I'll sign off. Do with this what you will.